File: VisualBasic/BasicAlgorithms/PrintTo255.vb
' Print all of the integers from 1 to 255.
Module PrintTo255

    Public Sub _For()
        Console.WriteLine("~~~ Begin For Loop -- 1-255 ~~~")
        For toPrint As Integer = 1 To 255
            Console.WriteLine(toPrint)
        Next
        Console.WriteLine("~~~ End For Loop -- 1-255 ~~~")
    End Sub 

    Public Sub _While()
        Dim toPrint As Integer = 0
        Console.WriteLine("~~~ Begin While Loop -- 1-255 ~~~")
        While toPrint <= 255
            Console.WriteLine(toPrint)
            toPrint += 1
        End While
        Console.WriteLine("~~~ End While Loop -- 1-255 ~~~")
    End Sub 

    Public Sub _DoWhile()
        Dim toPrint As Integer = 0
        Console.WriteLine("~~~ Begin Do While Loop -- 1-255 ~~~")
        Do While toPrint <= 255
            Console.WriteLine(toPrint)
            toPrint += 1
        Loop
        Console.WriteLine("~~~ End Do While Loop -- 1-255 ~~~")
    End Sub 
    
End Module 

 

File: Java/BasicAlgorithms/PrintTo255.java
package BasicAlgorithms;
/* Print all of the integers from 1 to 255.*/

public class PrintTo255
{
    public static void For()
    {
        System.out.println("~~~ Begin For Loop -- 1-255 ~~~");
        for (int toPrint = 1; toPrint < 256; toPrint++)
        {
            System.out.println(toPrint);
        }
        System.out.println("~~~ End For Loop -- 1-255 ~~~");
    }
    public static void PrintTo255_Do_While()
    {
        System.out.println("~~~ Begin Do While Loop -- 1-255 ~~~");
        int toPrint = 1;
        do {
            System.out.println(toPrint);
            toPrint++;
         } 
         while (toPrint < 256);
        System.out.println("~~~ End Do While Loop -- 1-255 ~~~");
    }
    public static void PrintTo255_While()
    {
        System.out.println("~~~ Begin While Loop -- 1-255 ~~~");
        int toPrint = 1;
        while (0 < toPrint && toPrint < 256)
        {
            System.out.println(toPrint);
            toPrint++;
        }
        System.out.println("~~~ End While Loop -- 1-255 ~~~");
    }
}